packages/Modal/tests/cypress/Modal.cypress.js

```diff
@@ -0,0 +1,14 @@
+import { getStoryUrlPrefix } from "../../../../.storybook/storyTree";
```

Comment:
created cypress tests as rtl won't be able to check dom tree with isDescendant helper

Reply:
Oh I thought RTL would not have the dom tree to navigate... 🤔

Reply:
it uses an in-memory DOM representation via https://github.com/jsdom/jsdom